Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:18.04
 
-RUN apt-get update && apt-get install -y --no-install-recommends curl cron awscli
+RUN apt-get update && apt-get install -y --no-install-recommends curl cron awscli less vim
```

Comment:
Why do you need `less` and `vim`?

Reply:
I suppose it was for debugging. Should probably be removed